pom.xml

```diff
@@ -432,6 +432,12 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+
+            <dependency>
```

Comment:
Nit: move this above to be after the `org.postgresql` dependency, since this is the Jersey section.